metabrainz/templates/index/team.html

```diff
@@ -0,0 +1,120 @@
+{% extends 'base.html' %}
+
+{% block title %}Team - MetaBrainz Foundation{% endblock %}
+
+{% block content %}
+<h1 class="page-title">Meet the MetaBrainz Foundation Team</h1>
+<div id="team" class="col-sm-12"> 
+ 
+  <div class="people">
+    <div class="pic long">
+      <img width="150px" height="150px" src="{{ url_for('static', filename='img/ruaok.jpg') }}">
```

Comment:
It also might be better to specify just one dimension (width), because if image is not square it will break.